test/functional/tests/cli/test_cli_standby.py

```diff
@@ -159,3 +162,78 @@ def test_activate_neg_cli_params():
                     f'Expected error message in format "{disallowed_param[0]}" '
                     f'Got "{output.stderr}" instead.'
                 )
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+def test_standby_neg_cli_management():
+    """
+    title: Blocking management commands in standby state
+    description: |
+      Try executing management commands for a cache in standby state
+    pass_criteria:
+      - The execution is unsuccessful for blocked management commands
+      - The execution is successful for allowed management commands
+      - A proper error message is displayed for unsuccessful executions
+    """
+    with TestRun.step("Prepare the device for the cache."):
+        device = TestRun.disks["cache"]
+        device.create_partitions([Size(500, Unit.MebiByte), Size(500, Unit.MebiByte)])
+        cache_device = device.partitions[0]
+        core_device = device.partitions[1]
+
+    with TestRun.step("Prepare the standby instance"):
+        cache_id = 1
+        cache = casadm.standby_init(
+            cache_dev=cache_device, cache_id=cache_id, cache_line_size=32, force=True
+        )
+
+        ioclass_config_path = "/tmp/standby_cli_neg_mngt_test_ioclass_config_file.csv"
+        random_ioclass_config = IoClass.generate_random_ioclass_list(5)
+        IoClass.save_list_to_config_file(
+            random_ioclass_config, ioclass_config_path=ioclass_config_path
+        )
+
+        blocked_mngt_commands = [
+            cli.get_param_cutoff_cmd(str(cache_id), "1"),
+            cli.get_param_cleaning_cmd(str(cache_id)),
+            cli.get_param_cleaning_alru_cmd(str(cache_id)),
+            cli.get_param_cleaning_acp_cmd(str(cache_id)),
+            cli.set_param_cutoff_cmd(str(cache_id), "1", threshold="1"),
+            cli.set_param_cutoff_cmd(str(cache_id), policy="never"),
+            cli.set_param_cleaning_cmd(str(cache_id), policy="nop"),
+            cli.set_param_cleaning_alru_cmd(str(cache_id), wake_up="30"),
+            cli.set_param_cleaning_acp_cmd(str(cache_id), wake_up="100"),
+            cli.set_cache_mode_cmd("wb", str(cache_id)),
+            cli.add_core_cmd(str(cache_id), core_device.path),
+            cli.remove_core_cmd(str(cache_id), "1"),
+            cli.remove_inactive_cmd(str(cache_id), "1"),
```

Comment:
add also remove core device from core pool command with flag --remove-detached

Reply:
If a core is in the core pool it means it is not added to any existing cache thus it can be removed at any time despite the cache being in the passive mode.

Reply:
ok, you are right here. :)
It does not affect standby state.